runtime/src/snapshot_config.rs

```diff
@@ -75,24 +75,39 @@ impl Default for SnapshotConfig {
 
 impl SnapshotConfig {
     /// A new snapshot config used for only loading at startup
-    #[must_use]
     pub fn new_load_only() -> Self {
         Self {
             usage: SnapshotUsage::LoadOnly,
             ..Self::default()
         }
     }
 
+    /// A new snapshot config used to disable snapshot generation and loading at
+    /// startup
+    pub fn new_disabled() -> Self {
+        Self {
+            usage: SnapshotUsage::Disabled,
+            ..Self::default()
+        }
+    }
+
     /// Should snapshots be generated?
-    #[must_use]
```

Comment:
Removed this because I wanted both `fn should_*` functions to be consistent and it's not necessary to force callers to use the result of a boolean method, they will obviously want to use it.